runtimes/swift.nix

```diff
@@ -0,0 +1,26 @@
+{pkgs, piston, ...}:
+let
+    pkg = pkgs.swift;
+in piston.mkRuntime {
+    language = "swift";
+    version = "5.4.2"; # pkg.version attribute is missing, so pinning it to 5.4.2
```

Comment:
somehow it shows up on search.nixos.org right, so it must be exported somehow.
Needs further investigation.

Reply:
My thoughts were the same on this

Reply:
was able figured it out after looking at the pkg source. apparently the version was not inherited and got baked into the name instead. I fixed it by using the parseDrvName builtin to extract the version from the name.